style.css

```diff
@@ -2,229 +2,780 @@
   /* stylelint-disable length-zero-no-unit */
   margin: 0px;
 
-  --font-family: 'poppins', sans-serif;
+  @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
+
+  text-decoration: none;
+  list-style-type: none;
+  box-sizing: border-box;
 }
 
 body {
-  margin: 0px;
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  overflow-x: hidden;
 }
 
-.section1 {
-  margin: 0px;
+/* start to homepage */
+.homepage-container {
+  display: grid;
+  grid-template-columns: 1349px;
+  justify-self: center;
+  height: 900px;
+  min-width: 1366px;
+  max-width: 100%;
+  background-color: #1c1a19;
+}
+
+.header {
   display: flex;
+  background-color: #3c3a39;
+  width: 1349px;
+  max-width: 100%;
+  height: 100px;
+  max-height: 100%;
+  z-index: 2;
+}
+
+.logo-text {
+  color: white;
+  margin-top: 30px;
+  margin-left: 150px;
+  font-family: 'Inter', sans-serif;
+  font-size: 26px;
+  font-weight: 900;
+  text-decoration: none;
+  display: inline-flex;
+  width: 100vh;
```

Comment:
- [ ] Kindly consider deleting this width, as it has no effect on your application. The width is measured in vw units instead of vh which is for the height.